### ISSUE TITLE
Knowledge Distillation Loss 

### ISSUE THREAD

**Issue by ghost** [NiccoBiondi/ContrastiveSupervisedDistillation#1]
Hi, such a nice and interesting work.

I've been taking a look to the code and I've a doubt about the Knowledge Distillation loss. What I noticed is that the knowledge is calculated with the logits of all the units, not just with the active units of the previously learned classes as it is done in Avalanche. Why do you propose to do it this way? If I'm not wrong, the Contrastive Supervised Distillation Loss is only calculated with the feature space of the old classes.

Thanks for the help and congrats for the great work.

**Comment by NiccoBiondi** [comment]
Hi Pablo, 
really thanks for your feedback.

Well, actually in our plugin KD loss is computed using only old classes logits as you can see from [this line](https://github.com/NiccoBiondi/ContrastiveSupervisedDistillation/blob/master/trainer/plugins.py#L69) and its following.

Let me know if I correctly understand your doubt.

Best!
Niccolò

**Comment by ghost** [comment]
Yeah, that's true. I focused on the SupConLoss and I missed that, everything seems fine.
